evm_loader/lib/src/commands/get_config.rs

```diff
@@ -57,11 +58,16 @@ pub enum ConfigSimulator<'r> {
 #[async_trait(?Send)]
 #[enum_dispatch]
 pub trait BuildConfigSimulator {
+    fn use_cache(&self) -> bool;
     async fn build_config_simulator(&self, program_id: Pubkey) -> NeonResult<ConfigSimulator>;
 }
 
 #[async_trait(?Send)]
 impl BuildConfigSimulator for CloneRpcClient {
+    fn use_cache(&self) -> bool {
+        true
```

Comment:
The `CHAINS_CACHE` logic guarded by this flag is only used in `CloneRpcClient`. Do you think `CHAINS_CACHE` could be moved directly into `ConfigSimulator::get_chains` method and `use_cache` be removed completely?

Reply:
We can't cache it for Clickhouse client. 
With clickhouse it can be requested for different slots and we can get different values for them.

Reply:
Caching separately for each slot is not what I want to do now.